### ISSUE TITLE
Permission to fork and publish

### ISSUE THREAD

**Issue by sergiooak** [spiralofhope/SuperDuperMacro#38]
Hey, I've made a fork and made a few tweaks (I only found your fork after that)...
So I have done the things the right way, forked and updated to keep yours and hypehuman's credits

I tried to contact hypehuman, but there are few years that he don't seem to be active anywhere.
Can you grant me permission to post and keep updating on Curse Forge?

**Comment by Wraithlord2015** [comment]
Yes - pls and tks in advance! This addon very much needs an update!
